breacharbiter.go

```diff
@@ -625,6 +645,63 @@ Loop:
 
 			goto justiceTxBroadcast
 
+		// On every new block, we check whether we should republish the
+		// transactions.
+		case <-newBlockChan.Epochs:
+			blocksPassed++
+
+			// If less than four blocks have passed since we
+			// published the justice tx, we'll continue waiting. It
+			// was published with a 2-block fee estimate, so it's
+			// not unexpected that four blocks without confirmation
+			// can pass.
+			if blocksPassed < 4 {
```

Comment:
note that we may never pass this check on mobile, since lnd needs to observe 4 block consecutively without restarting. not sure what the best way to handle that is, is there a way we can load the broadcast height on startup?

Reply:
Hmm, yeah this is a good point...since we no longer save the finalized transaction, we can't use the locktime there to mark our last broadcast height. We do have the `CloseHeight` as part of the close summary though, which tracks the height the breaching transaction was confirmed.  I think that should work as a replacement?